pyveda/frameworks/batch_generator.py

```diff
@@ -12,13 +14,17 @@ class BaseGenerator():
     Rescale: Boolean. Flag to indicate if data returned from the generator should be rescaled between 0 and 1.
     '''
 
-    def __init__(self, cache, batch_size=32, shuffle=True, rescale=False):
+    def __init__(self, cache, batch_size=32, shuffle=True, rescale=False,
+                    flip_horizontal=False, flip_vertical=False):
         self.cache = cache
         self.batch_size = batch_size
         self.index = 0
         self.shuffle = shuffle
         self.rescale = rescale
         self.on_epoch_end()
+        self.flip_h = flip_horizontal
+        self.flip_v = flip_vertical
+        self._applied_augs(self.flip_h, self.flip_v)
         self.list_ids = np.arange(0, len(self.cache))
```

Comment:
`np.arange` takes a number and makes a range, so 0 is redundant: `np.arange(len(self.cache))`

Reply:
fixed